lambdas/data-migration1/src/rules.ts

```diff
@@ -114,7 +114,7 @@ export const migrateRules = async (
       } else {
         migrationSummary.failed += 1;
         logger.error(
-          `Could not create rule record in RDS for Dynamo Rule name: ${record.name}, version: ${record.version}}`,
+          `Could not create rule record in RDS for Dynamo Rule name: ${record.name}, version: ${record.version}`,
```

Comment:
This looks like a bad copy/paste from the collections migration. Dynamo rules don't have a `version`